lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -16,6 +16,17 @@
  */
 class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_5_9 {
 
+	const ELEMENTS = array(
+		'link' => 'a',
+		'h1'   => 'h1',
+		'h2'   => 'h2',
+		'h3'   => 'h3',
+		'h4'   => 'h4',
+		'h5'   => 'h5',
+		'h6'   => 'h6',
+		'button' => '.wp-element-button',
```

Comment:
Should probably move this change to the 6.1 folder now and rename the class on this file to `WP_Theme_JSON_6_0`

Reply:
Done